front/src/reducers/osrdconf/osrdConfCommon/utils.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/prefer-default-export */
+
+import type { PathStep } from '../types';
+
+export const checkValidPathStep = (pathStep: PathStep) => {
+  if (
+    !pathStep ||
+    pathStep.locked ||
+    pathStep.arrival ||
+    pathStep.stopFor ||
+    pathStep.theoreticalMargin
+  )
+    return false;
+  return true;
+};
```

Comment:
I think you can reformat this function like that. But its strange to return false to a function called "checkValidPathStep" if we have one of its property "truthy" don't you think ?

```suggestion
export const checkValidPathStep = (pathStep: PathStep) => 
  pathStep &&
  !pathStep.locked &&
  !pathStep.arrival &&
  !pathStep.stopFor &&
  !pathStep.theoreticalMargin;

```

Reply:
Done, the function has been renamed "canRemovePathStep" ;)